src/asana/helpers.py

```diff
@@ -394,22 +398,26 @@ def _task_description_from_pull_request(pull_request: PullRequest) -> str:
         + "\n✍️ "
         + author
         + _generate_assignee_description(pull_request.assignee())
+        + f"\n❗️Task is {status} because {status_reason.reason}"
         + _wrap_in_tag("strong")("\n\nDescription:\n")
         + convert_urls_to_links(escape(pull_request.body(), quote=False))
     )
 
 
 def _task_completion_from_pull_request(pull_request: PullRequest) -> bool:
```

Comment:
```suggestion
def _task_completion_from_pull_request(pull_request: PullRequest) -> StatusReason:
```

Reply:
Can you run `black` and `mypy` on the `src` and `test` directories to make sure both pass?